src/script/lua_api/l_server.cpp

```diff
@@ -42,6 +42,18 @@ int ModApiServer::l_request_shutdown(lua_State *L)
 	return 0;
 }
 
+int ModApiServer::l_transfer_player(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	const char *playerName = lua_tolstring(L, 1, NULL);
```

Comment:
Wouldn't `readParam<std::string>` work?

Reply:
Probably will work. I have rebased it, not rewritten it.